doc/user/content/transform-data/json.md

```diff
@@ -71,6 +71,10 @@ function debounce(callback, wait) {
     }
 }
 
+function camelToSnakeCase(s) {
+    return s.split(/(?=[A-Z])/).map(word => word.toLowerCase()).join("_");
```

Comment:
Do we have access to a library for this? Like Lodash, which offers `_.snakeCase`? This implementation won't handle initialisms/acronyms properly:

![image](https://github.com/MaterializeInc/materialize/assets/882976/73094672-b6ad-4535-9366-a194e61fd090)

Reply:
I had wanted to add `lodash` to replace a few other util functions I added, just put up https://github.com/MaterializeInc/materialize/pull/23839 to do this.